src/appReducer.js

```diff
@@ -0,0 +1,84 @@
+import { getUKFormattedDate } from './utils/helpers';
+
+const appReducer = (state, action) => {
+  switch (action.type) {
+    case 'RESTORE_EXPENSES':
```

Comment:
question: why not use Actions constants? Is it because they are anyway only used in `GlobalState.js`?

Reply:
I opted to pass down callbacks that trigger the dispatches (e.g. `addExpense`, `closeConfirmMonthModal`) from `GlobalState.js` instead of dispatching from lower in the component tree. Does this make sense?